### PR TITLE
Add more tests

### LINKED ISSUE #180: Bug: Buyer is not being added to listing
image

### PR DESCRIPTION
**Pull request by e1121208**
Fix #180 concurrently

### CODE REVIEW

File: src/test/java/seedu/address/logic/commands/AddBuyersToListingCommandTest.java
Comment (lrongyi) [inline src/test/java/seedu/address/logic/commands/AddBuyersToListingCommandTest.java:1]: LGTM, do add checks if the listing is invalid, or the listing or buyer does not exist

File: src/test/java/seedu/address/logic/commands/AddBuyersToListingCommandTest.java [hunk 1]
```diff
@@ -0,0 +1,325 @@
...
+
+    @Test
+    public void constructor_nullPerson_throwsNullPointerException() {
+        assertThrows(NullPointerException.class, () -> new AddBuyersToListingCommand(VALID_LISTING_NAME,
+                null));
+    }
+
+    /*
+    @Test
+    public void constructor_nullListing_throwsCommandException() throws CommandException {
```
Comment (lrongyi) [inline src/test/java/seedu/address/logic/commands/AddBuyersToListingCommandTest.java:63]: This should be tested as well

File: src/test/java/seedu/address/logic/commands/ClearListingCommandTest.java
Comment (lrongyi) [inline src/test/java/seedu/address/logic/commands/ClearListingCommandTest.java:1]: LGTM

File: src/test/java/seedu/address/logic/commands/DeleteListingCommandTest.java [hunk 2]
```diff
@@ -0,0 +1,134 @@
...
+import seedu.address.model.listing.Address;
+import seedu.address.model.listing.Area;
+import seedu.address.model.listing.Listing;
+import seedu.address.model.listing.Price;
+import seedu.address.model.listing.Region;
+import seedu.address.model.person.Name;
+import seedu.address.testutil.PersonBuilder;
+
+public class DeleteListingCommandTest {
+    private static final Listing WOODLANDS_LISTING = new Listing(
```
Comment (lrongyi) [inline src/test/java/seedu/address/logic/commands/DeleteListingCommandTest.java:29]: Do not need to redeclare, can use TypicalListings

File: src/test/java/seedu/address/logic/commands/FindListingCommandTest.java [hunk 3]
```diff
@@ -0,0 +1,133 @@
...
+import seedu.address.model.listing.Area;
+import seedu.address.model.listing.Listing;
+import seedu.address.model.listing.ListingContainsKeywordsPredicate;
+import seedu.address.model.listing.Price;
+import seedu.address.model.listing.Region;
+import seedu.address.model.person.Name;
+import seedu.address.testutil.PersonBuilder;
+
+public class FindListingCommandTest {
+    private static final Listing WOODLANDS_LISTING = new Listing(
```
Comment (lrongyi) [inline src/test/java/seedu/address/logic/commands/FindListingCommandTest.java:31]: Do not need to redeclare, can use TypicalListing

File: src/test/java/seedu/address/logic/commands/AddBuyersToListingCommandTest.java (hunk 1 above)
Comment (e1121208) [reply src/test/java/seedu/address/logic/commands/AddBuyersToListingCommandTest.java:63]: sorry love, im not strong enough to do it

**Pull request merged**